src/main/views/MattermostView.test.js

```diff
@@ -1,9 +1,374 @@
 // Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+'use strict';
+
+import {LOAD_FAILED, TOGGLE_BACK_BUTTON, UPDATE_TARGET_URL} from 'common/communication';
+import {MattermostServer} from 'common/servers/MattermostServer';
+import MessagingTabView from 'common/tabs/MessagingTabView';
+
+import * as WindowManager from '../windows/windowManager';
+import * as appState from '../appState';
+import Utils from '../utils';
+
+import {MattermostView} from './MattermostView';
+
+jest.mock('electron', () => ({
+    app: {
+        getVersion: () => '5.0.0',
+    },
+    BrowserView: jest.fn().mockImplementation(() => ({
+        webContents: {
+            loadURL: jest.fn(),
+            on: jest.fn(),
+            getTitle: () => 'title',
+            getURL: () => 'http://server-1.com',
+        },
+    })),
+    ipcMain: {
+        on: jest.fn(),
+        handle: jest.fn(),
+    },
+}));
+
+jest.mock('../windows/windowManager', () => ({
+    sendToRenderer: jest.fn(),
+    focusThreeDotMenu: jest.fn(),
+}));
+jest.mock('../appState', () => ({
+    updateMentions: jest.fn(),
+}));
+jest.mock('./webContentEvents', () => ({
+    removeWebContentsListeners: jest.fn(),
+}));
+jest.mock('../contextMenu', () => jest.fn());
+jest.mock('../utils', () => ({
+    getLocalURLString: jest.fn(),
+    getWindowBoundaries: jest.fn(),
+    getLocalPreload: (file) => file,
+    composeUserAgent: () => 'Mattermost/5.0.0',
+    shouldHaveBackBar: jest.fn(),
+}));
+
+const server = new MattermostServer('server_name', 'http://server-1.com');
+const tabView = new MessagingTabView(server);
+
 describe('main/views/MattermostView', () => {
-    // Jest won't stop yelling at me for not having a test in here
-    it('should do nothing', () => {
-        expect(1).toBe(1);
+    describe('load', () => {
+        const window = {on: jest.fn()};
+        const mattermostView = new MattermostView(tabView, {}, window, {});
+
+        beforeEach(() => {
+            Utils.getLocalURLString.mockReturnValue('file://some/file/path/mattermost.html');
+            mattermostView.loadSuccess = jest.fn();
+            mattermostView.loadRetry = jest.fn();
+        });
+
+        // it('should load provided URL when provided', async () => {
```

Comment:
(question) do we intend to bring back the tests, so this is why they're commented (a temporary thing)? If not, let's remove the test cases altogether.

Reply:
I need to re-evaluate how the `load()` function is going to work going forward, because we'll always be loading relative to the local app. I'm going to leave them commented for now, and if possible I'd like to bring them back or modify them.